### PR TITLE
#6 - Upgraded http module to 0.5

### PR DESCRIPTION
**Pull request by olegmoz**
Closes issue #6

### CODE REVIEW

**Comment by paulodamaso** [comment]
@g4s8 I'm available, please assign me

**Comment by 0crat** [comment]
This pull request [#7](https://github.com/artipie/files-adapter/issues/7) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/3fc5acd4-c589-45c5-840d-f2ddd71601ac); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by 0crat** [comment]
Job [#7](https://github.com/artipie/files-adapter/issues/7) is already in scope

File: src/main/java/com/artipie/files/FilesSlice.java [hunk 1]
```diff
@@ -65,40 +65,31 @@ public FilesSlice(final Storage storage) {
...
-                    new HashSet<>(zero),
-                    FlowAdapters.toFlowPublisher(
-                        Flowable.fromPublisher(reactive)
-                            .flatMapCompletable(byteBuffer -> Completable.complete())
-                            .andThen(this.storage.value(key).flatMapPublisher(flow -> flow))
-                    )
-                );
-            };
-        } else if (method.equals("POST") || method.equals("PUT")) {
+        if (method.equals(RqMethod.GET)) {
```
Comment (paulodamaso) [inline src/main/java/com/artipie/files/FilesSlice.java]: @olegmoz You could use a `switch` instead `if-elseif-else` statements here, the JVM generates a more efficient bytecode when using `switch`

Comment (olegmoz) [reply src/main/java/com/artipie/files/FilesSlice.java]: @paulodamaso not really in the scope of the issue, but really makes sense. Made the changes

**Comment by paulodamaso** [comment]
@g4s8 Seems good to me, please merge.

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR

**Comment by 0crat** [comment]
Job was finished in 20 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Pull request merged**